x-pack/packages/security-solution/navigation/src/landing_links/landing_links_image_card.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiText, EuiTitle, useEuiTheme } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { css } from '@emotion/react';
+import { withLink } from '../links';
+import type { NavigationLink } from '../types';
+import { BetaBadge } from './beta_badge';
+import { getKibanaLinkProps } from './utils';
+
+export interface LandingLinksImageCardProps {
+  item: NavigationLink;
+  urlState?: string;
+  onLinkClick?: (id: string) => void;
+}
+
+const CARD_HEIGHT = 116;
+const CARD_WIDTH = 370;
+const CARD_HEIGHT_IMAGE = 98;
+
+const useStyles = () => {
+  const { euiTheme } = useEuiTheme();
+  return {
+    card: css`
+      height: ${CARD_HEIGHT}px;
+      max-width: ${CARD_WIDTH}px;
+    `,
+    cardWrapper: css`
+      height: 100%;
+    `,
+    titleContainer: css`
+      height: ${euiTheme.size.l};
+    `,
+    title: css`
+      color: ${euiTheme.colors.primaryText};
+      font-weight: ${euiTheme.font.weight.semiBold};
+    `,
+    getImageContainer: (imageUrl: string | undefined) => css`
+      overflow: hidden;
+      height: ${CARD_HEIGHT_IMAGE}px;
+      width: ${CARD_HEIGHT_IMAGE}px;
+      flex-grow: 0;
+      background-position: center center;
+      background-repeat: no-repeat;
+      background-image: url(${imageUrl ?? ''});
+      background-size: auto 98px;
+    `,
```

Comment:
A great solution to use background image 👍 NIT: it makes `overflow: hidden;` and `flex-grow: 0;` unnecessary.